parallel-install/pkg/deployment/deployment.go

```diff
@@ -229,7 +232,7 @@ func (i *Deployment) deployPrerequisites(ctx context.Context, cancelFunc context
 		Context:       ctx,
 		KubeClient:    i.kubeClient,
 		Prerequisites: p,
-		Log:           i.cfg.Log,
+		Log:           logger.NewLogger(i.cfg.Verbose),
```

Comment:
Do we really need to instantiate a separate logger for every component? Is that actually safe? See: https://github.com/uber-go/zap/issues/99

Reply:
After followup, we have decided to change the strategy. Now logger will be created once on the highest level and passed around.